virtme/mkinitramfs.py

```diff
@@ -14,6 +14,32 @@
 import itertools
 from . import cpiowriter
 from . import util
+from elftools.elf.elffile import ELFFile
+from elftools.elf.dynamic import DynamicSection
+
+ld_paths = ['/lib', '/lib65', '/usr/lib', '/usr/lib64']
```

Comment:
`lib65` here looks like a typo...

Reply:
Indeed. `ld-musl-x86_64.so.1` was conveniently placed in `/lib` to avoid this :)